src/sql/workbench/services/objectExplorer/browser/objectExplorerService.ts

```diff
@@ -517,6 +518,22 @@ export class ObjectExplorerService implements IObjectExplorerService {
 			}
 
 			finalResult.nodes = allNodes;
+			if (allNodes.length === 0) {
+				const expansionReturnedNoNodesDisplayString = nls.localize('nodeExpansionNoNodes', "Expansion returned no nodes");
```

Comment:
Is there a reason you're adding this here? Is there feedback about this behavior?

I would suggest doing this as a separate PR so it can be discussed separately there since I'm not sure about changing this behavior given that neither SSMS nor the VS Code MSSQL extension have this behavior currently.

Reply:
Reverting for this PR